keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets.py

```diff
@@ -40,6 +43,9 @@
                 "[1, 3, 3, 1] depths where the batch normalization "
                 "and SiLU activation are applied after the convolution layers."
             ),
+            "params": 4223488,
```

Comment:
perhaps we should have task metadata to embed in the table as well!

Reply:
I assume luke means add a list of `Task` subclasses, which the backbone class is compatible with, as a field of the metadata. Not all backbones compatible with all tasks?